SQRLDotNetClientUI/ViewModels/MessageBoxViewModel.cs

```diff
@@ -159,4 +171,29 @@ public async Task<MessagBoxDialogResult> ShowDialog(ViewModelBase parent)
             });
         }
     }
+
+    /// <summary>
+    /// Custom button message class to pass in to our MessageBox
+    /// </summary>
+    public class  MessageBoxButtonCustom
```

Comment:
Wouldn't `MessageBoxCustomButton` be a bit more straigthforward here?

Reply:
yeah no I like my confusing backwards class names thank you very much 😋 (fixing it LoL)